docs/guide/Titanium_SDK/Titanium_SDK_How-tos/User_Interface_Deep_Dives/iOS_UI_Components_and_Conventions/README.md

```diff
@@ -133,7 +133,7 @@ With 1.8, the location of the Settings.bundle was changed. Platform-specific res
 
 If you're running SDK 1.8+
 
-1. Download a recent copy of the [KitchenSink app](https://github.com/appcelerator/KitchenSink)
+1. Download a recent copy of the [KitchenSink app](https://github.com/tidev/KitchenSink)
```

Comment:
Bad link. The correct link is https://github.com/tidev/kitchensink-v2. There are multiple instances, so grep to find them all. Thanks!

Reply:
7 replacements in 4 files